visualize.py

```diff
@@ -6,11 +6,11 @@
 """
 from train import *
 
-saver = tf.train.import_meta_graph('model.meta')
+saver = tf.train.Saver()
```

Comment:
By using default saver, I found it could avoid Error, although I do not know the reason.